### ISSUE TITLE
Option to change between dark theme/white theme

### ISSUE THREAD

**Issue by ooHAoo** [yevseytsev/SenecaBlackboardExtension#98]
Please correct me if I'm wrong but I thought the default is the white/red theme (the normal) and the users have the option to change to dark theme. Don't see that option anywhere and I think that would better as some may prefer the white theme and some the dark theme.

If this isn't already done then I will work on it 
(but I thought it's already done somewhere, just don't see it)

**Comment by Casva** [comment]
@ooHAoo That's correct, the default theme should be white with an option to switch to other themes. I submitted a pull request for an options panel for the extension over two weeks ago but it only has recently been reviewed and needs to be rebased. 

My idea was for the themes to be included in the options as well, but adding a toggle switch / button on the website itself could work as well. Let me know what you think!

**Comment by yuzhouChen** [comment]
I'm working on the switch version button right now, but right now I'm only able to add all useless content back. I think it's good to have another button for change the theme. Then the switch button will only add or remove useless information from the Seneca Blackboard

**Comment by Casva** [comment]
@yuzhouChen the idea of the options panel (issue #48) was to allow for the users to select which features to implement. This includes removing specific components (e.g. only remove Seneca spotlight and not Seneca news) and also implementing changes to existing components. It makes it more convenient for users to fully customize their preferences. 

The button for switching themes can go either way. Since there are only 2 additional themes made so far, it does seem viable. A problem would arise if more themes keep getting added since a button wouldn't be as viable to select a specific one.

Ultimately it depends on whether or not the option panel wants to be implemented. That's just my opinion! 😃

**Comment by ooHAoo** [comment]
@yuzhouChen @Casva 
Generally speaking (or based on a user's point of view) I feel like there shouod really only be two themes, dark and white(bright) since the purpose of these themes are for vision purposes.
I know many that would rather have the white/normal theme and having the dark theme as an option is nice too. So I think having the normal theme as the default and giving the users the option to click on a button and have the extension change to a dark theme.

Sorry if I'm making it confusing lol but in short all I'm trying to say is to have a button to change between the white and dark theme would be nice, while having the white theme as the default for our extension.  (But this is just my preference, it all depends on what the majority thinks, of course)
